### PR TITLE
Update docs for google analytics

### PR DESCRIPTION
**Pull request by Salonijain27**
Updates _config.yml file and adds _include/analytics.html file to set up google analytics tracker

### CODE REVIEW

File: _include/analytics.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<head>
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-203084925-1"></script>
```
Comment (sameerz) [inline _include/analytics.html:3]: Please confirm the UA and GA tags are connected.

**Review by sameerz — APPROVED** [review]

File: _include/analytics.html (hunk 1 above)
Comment (Salonijain27) [reply _include/analytics.html]: Updated the code to include only GA tags

**Comment by sameerz** [comment]
One more request - can you retarget this to 21.08, and then we can cherry pick this into the gh-pages branch?

**Comment by Salonijain27** [comment]
Re-creating the PR with 21.08 as its base

**Pull request closed without merging**